pkg/cmd/run/shared/shared.go

```diff
@@ -42,6 +43,19 @@ type Status string
 type Conclusion string
 type Level string
 
+var RunFields = []string{
```

Comment:
What about some other fields that might be useful to get [for a workflow run](https://docs.github.com/en/rest/reference/actions#get-a-workflow-run)?
- `head_branch`
- `head_sha`
- `event`
- `status`